index.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: default
+title: Open Containers
+nav_order: 1
+permalink: /
+---
+
+# The Open Container Initiative
```

Comment:
Super nit: I think "Open Container Initiative" is canonical, but I see "Open Containers Initiative" a lot in this PR.

Reply:
hmm so which one should we use? The oci site says "Open Container Initiative" but then there are subtle details, like the site name being "opencontainers.org." I think I embarrassingly have used them both! Which one do you prefer / is correct?

Reply:
Was just correcting this in another publication recently; it is most definitely the "Open Container Initiative" (see the project's [about page](https://www.opencontainers.org/about)). Agree that the domain name may add to the confusion for newcomers!

Reply:
okay, all are fixed.